README.md

```diff
@@ -126,6 +126,10 @@ It is **not** necessary to specify values for all configurable properties. _Only
     "submitLabel" : "Next",
     "showSearchAgainLink" : false,
     "searchAgainLinkText" : "Search again"
+  },
+  "timeout" : {
+    "timeoutAmount" : 900,
+    "timeoutUrl" : "http://service/timeout-uri"
```

Comment:
This doesn't look to me like any kind of legitimate default value. If there is no sensible default, then we should make it a required piece of configuration, as we do with the continue URL. Basically, we are demanding that calling services implement some kind of endpoint here, aren't we? They will *have* to tell us where it is.

Reply:
Hi Chris, there is no default value for the timeout block, I mentioned it in the Table below

Reply:
Sorry, yes - I am being dumb. This is just the readme!